library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_instance.hpp

```diff
@@ -13,6 +13,7 @@ namespace tensor_operation {
 namespace device {
 namespace instance {
 
+using F8   = ck::f8_t;
```

Comment:
Could you add "#ifdef CK_ENABLE_FP8" like we do here https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/58817bf967d3b7b314233f6331f0430cf40651ae/library/include/ck/library/tensor_operation_instance/device_operation_instance_factory.hpp#L23